protostar/commands/test/test_case_runners/fuzz_test_case_runner.py

```diff
@@ -8,27 +8,27 @@
     FuzzInputExceptionMetadata,
 )
 from protostar.commands.test.test_case_runners.test_case_runner import TestCaseRunner
+from protostar.commands.test.test_environment_exceptions import ReportedException
 from protostar.commands.test.test_results import (
     FailedFuzzTestCaseResult,
     FailedTestCaseResult,
     FuzzResult,
     PassedFuzzTestCaseResult,
     TestResult,
 )
-from protostar.commands.test.test_environment_exceptions import ReportedException
 
 
 class FuzzTestCaseRunner(TestCaseRunner[FuzzTestExecutionResult]):
     def __init__(
-        self,
-        fuzz_test_execution_environment: FuzzTestExecutionEnvironment,
-        test_case_runner_deps: TestCaseRunner.Dependencies,
+        self, fuzz_test_execution_environment: FuzzTestExecutionEnvironment, **kwargs
     ) -> None:
-        super().__init__(test_case_runner_deps)
+        super().__init__(**kwargs)
```

Comment:
typechecker can't verify kwargs

Reply:
I've got a feeling I'm going to complain in the next following PR. Anyway, resolving.